tests/Feature/UserSettingsTest.php

```diff
@@ -355,7 +355,7 @@
     $response->assertOk();
 });
 
```

Comment:
Please add tests to "users can edit roles and permissions" test to check that:

- [ ] Organization users with an organization can access the page
- [ ] Regulated organization users without a regulated organization cannot access the page
- [ ] Organization users without an organization cannot access the page